client/state/action-types.js

```diff
@@ -6,6 +6,8 @@
  */
 
 export const FAIL_PUBLICIZE_CONNECTIONS_REQUEST = 'FAIL_PUBLICIZE_CONNECTIONS_REQUEST';
+export const FETCH_SITE_PLANS = 'FETCH_SITE_PLANS';
+export const FETCH_SITE_PLANS_COMPLETED = 'FETCH_SITE_PLANS_COMPLETED';
```

Comment:
What I mean is that the correct order for these actions should be:

```

FETCH_PUBLICIZE_CONNECTIONS
FETCH_SITE_PLANS
FETCH_SITE_PLANS_COMPLETED
RECEIVE_PUBLICIZE_CONNECTIONS
``` 
but this order isn't very useful.